### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by viren-nadkarni**
Removes the bit about PyPI install as it's not yet published there

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,19 @@ A command line interface is also included for convenience.
...
+
     $ pip install speechmatics-python
 
-To install from source:
+-->
+
+To install:
 
-    $ python setup.py install
+    $ git clone https://github.com/speechmatics/speechmatics-python
```
Comment (weakcamel) [inline README.md:22]: perhaps https://packaging.python.org/tutorials/installing-packages/#id23 ?

File: README.md [hunk 2]
```diff
@@ -8,13 +8,19 @@ A command line interface is also included for convenience.
 
 ## Getting started
 
+<!--
 To install from PyPI:
 
+PIP EXTRA URL
+
     $ pip install speechmatics-python
```
Comment (weakcamel) [inline README.md:16]: `pip install --extra-index-url https://test.pypi.org/legacy/ speechmatics-python` ?

File: README.md (hunk 1 above)
Comment (viren-nadkarni) [reply README.md:20]: Good one, fixed 👍

File: README.md (hunk 2 above)
Comment (viren-nadkarni) [reply README.md:14]: Removed that altogether (this is to be uncommented when this is finally published to PyPI)

Comment (weakcamel) [reply README.md:14]: fair enough; btw - anything we want to do before we start publishing to pypi.org?

Comment (viren-nadkarni) [reply README.md:14]: Waiting for green light from Product

Comment (weakcamel) [reply README.md:14]: @viren-nadkarni 👍

**Pull request merged**